### DISCUSSION TITLE
Problem with long cable

### DISCUSSION THREAD

**Discussion by ltarmcool** [RudolphRiedel/FT800-FT813#84] [Q&A] [answered]
Hi Rudolph,
Thank you for your library, I was able to use it with a PIC24F microcontroller to display on Riverdi LCD RVT43HLTFWN00. I have to use a buffer chip (TXS0108EPWR, without this chip I couldn't even get it to work) for SPI signals and to connect between my PCB and their LCD screen I use 20 Position FFC, FPC Connector and FFC cable (~150mm). Everything works well, but I want to increase the cable length and notice that it works until ~200mm, then the SPI signals degraded badly and stopped working. I wondered if you have seen something similar, I run it at 4MHz SPI clock
Regards,

Lam

**Comment by RudolphRiedel** [comment, marked as answer]
Hello,
I have not really tried to find out what length of FFC cable works and I usually use rather short cables of 50mm since I put my controllers boards with the display module in a case.

When I am using eval boards or all kinds of Arduino boards I am using one my adapter boards though:
https://github.com/RudolphRiedel/EVE_display-adapter/tree/master/L-D5019-08

And to these I connect with jumper cables of 10cm...20cm.
I use 8MHz to 16MHz regulary without issue.

What I took from these boards over a couple of iterations is that a plain series resistor allows the most boards to play nice with the displays and so I finally ended up using 56R.

**Reply by ltarmcool** [reply]
Hi Rudolph, thank you for your quick answer, I will tried it with a series resistor and give you update
